packages/xstate-immer/package.json

```diff
@@ -36,7 +36,7 @@
   },
   "dependencies": {},
   "peerDependencies": {
-    "immer": "^2.1.5",
+    "immer": "^3.2.0",
```

Comment:
we have 3.2.0 in devDeps, so I've aligned them - not sure if this is completely correct now though, this package is not published yet from what I remember so it doesn't matter much i think